opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/aggregation.py

```diff
@@ -1387,18 +1387,17 @@ def _create_aggregation(
                 AggregationTemporality.CUMULATIVE
             )
 
-        if self._boundaries is None:
-            self._boundaries = (
-                instrument._advisory.explicit_bucket_boundaries
-                or _DEFAULT_EXPLICIT_BUCKET_HISTOGRAM_AGGREGATION_BOUNDARIES
-            )
+        if self._boundaries:
```

Comment:
```suggestion
        if self._boundaries is not None:
```

Not sure passing empty boundaries is correct but I think this should check the default None